### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Sikirash** [aa2g/AA2Unlimited#184]
Greetings,
I often get crashes and just want to know if its related to aau or something else.

[crash.zip](https://github.com/aa2g/AA2Unlimited/files/3692640/crash.zip)

**Comment by k4yf0ur** [comment]
Unfortunately the crash dump isn't informative enough to tell. I'd need more information about your game, and when the crashes occur.

**Comment by Sikirash** [comment]
Thanks for responding :)

The game has some light modding like hexa packs and more cumming positions and that's about it.

As for the info on when it happens....it's a tricky one. For sure it happens way more often with more characters and it does happen more when I interact or when other npcs try to interact. But it is not consistent enough for me to pinpoint problem. Interaction between characters is problem for sure because it never crashed when I am alone (everyone went home). 

At first I thought certain heavy modcards were problem but it's just not crashing every time for me to conclude anything :/

So, to summarize my impressions about this problem:
- crashes occasionally when trying to talk
-                               when npcs try to talk
- happens more often with more npcs (obviously)
- seems that increased game speed amplifies the effect    

with such "unreliable" symptoms I doubt something will be resolved but thanks nonetheless

**Comment by linkanon** [comment]
from my experience there's some crashes that are triggered by opening the class roster and generally fiddling with different menus. Try looking out for those as well and see if there's a pattern related to them
